2022-03-14 16:02:40,973 INFO    BackendThr:7212 [internal.py:wandb_internal():90] W&B internal server running at pid: 7212, started at: 2022-03-14 16:02:40.972553
2022-03-14 16:02:40,973 INFO    MainThread:7212 [wandb_init.py:init():469] backend started and connected
2022-03-14 16:02:40,974 DEBUG   MainThread:7212 [config_util.py:dict_from_config_file():102] no default config file found in config-defaults.yaml
2022-03-14 16:02:40,975 INFO    MainThread:7212 [wandb_init.py:init():533] updated telemetry
2022-03-14 16:02:40,977 INFO    WriterThread:7212 [datastore.py:open_for_write():77] open: D:\context_enhancement\context-main\combined\context_enhancement\wandb\run-20220314_160240-1u15e8i3\run-1u15e8i3.wandb
2022-03-14 16:02:40,978 DEBUG   SenderThread:7212 [sender.py:send():234] send: header
2022-03-14 16:02:41,024 INFO    MainThread:7212 [wandb_init.py:init():563] communicating current version
2022-03-14 16:02:41,024 DEBUG   HandlerThread:7212 [handler.py:handle_request():130] handle_request: check_version
2022-03-14 16:02:41,025 DEBUG   SenderThread:7212 [sender.py:send_request():248] send_request: check_version
2022-03-14 16:02:41,327 INFO    MainThread:7212 [wandb_init.py:init():568] got version response upgrade_message: "wandb version 0.12.11 is available!  To upgrade, please run:\n $ pip install wandb --upgrade"

2022-03-14 16:02:41,328 INFO    MainThread:7212 [wandb_init.py:init():578] communicating run to backend with 30 second timeout
2022-03-14 16:02:41,329 DEBUG   SenderThread:7212 [sender.py:send():234] send: run
2022-03-14 16:02:41,743 INFO    MainThread:7212 [wandb_init.py:init():606] starting run threads in backend
2022-03-14 16:02:41,744 DEBUG   HandlerThread:7212 [handler.py:handle_request():130] handle_request: run_start
2022-03-14 16:02:41,745 INFO    SenderThread:7212 [dir_watcher.py:__init__():169] watching files in: D:\context_enhancement\context-main\combined\context_enhancement\wandb\run-20220314_160240-1u15e8i3\files
2022-03-14 16:02:41,745 INFO    SenderThread:7212 [sender.py:_start_run_threads():807] run started: 1u15e8i3 with start time 1647253960
2022-03-14 16:02:41,745 DEBUG   SenderThread:7212 [sender.py:send():234] send: summary
2022-03-14 16:02:41,745 INFO    SenderThread:7212 [sender.py:_save_file():939] saving file wandb-summary.json with policy end
2022-03-14 16:02:41,759 DEBUG   HandlerThread:7212 [meta.py:__init__():40] meta init
2022-03-14 16:02:41,759 DEBUG   HandlerThread:7212 [meta.py:__init__():54] meta init done
2022-03-14 16:02:41,759 DEBUG   HandlerThread:7212 [meta.py:probe():214] probe
2022-03-14 16:02:41,776 DEBUG   HandlerThread:7212 [meta.py:_setup_git():204] setup git
2022-03-14 16:02:41,874 DEBUG   HandlerThread:7212 [meta.py:_setup_git():211] setup git done
2022-03-14 16:02:41,874 DEBUG   HandlerThread:7212 [meta.py:_save_pip():58] save pip
2022-03-14 16:02:41,875 DEBUG   HandlerThread:7212 [meta.py:_save_pip():72] save pip done
2022-03-14 16:02:41,875 DEBUG   HandlerThread:7212 [meta.py:_save_conda():79] save conda
2022-03-14 16:02:42,752 INFO    Thread-17 :7212 [dir_watcher.py:_on_file_created():217] file/dir created: D:\context_enhancement\context-main\combined\context_enhancement\wandb\run-20220314_160240-1u15e8i3\files\wandb-summary.json
2022-03-14 16:02:42,753 INFO    Thread-17 :7212 [dir_watcher.py:_on_file_created():217] file/dir created: D:\context_enhancement\context-main\combined\context_enhancement\wandb\run-20220314_160240-1u15e8i3\files\conda-environment.yaml
2022-03-14 16:02:42,753 INFO    Thread-17 :7212 [dir_watcher.py:_on_file_created():217] file/dir created: D:\context_enhancement\context-main\combined\context_enhancement\wandb\run-20220314_160240-1u15e8i3\files\requirements.txt
2022-03-14 16:02:44,009 DEBUG   HandlerThread:7212 [meta.py:_save_conda():89] save conda done
2022-03-14 16:02:44,010 DEBUG   HandlerThread:7212 [meta.py:probe():252] probe done
2022-03-14 16:02:44,014 DEBUG   SenderThread:7212 [sender.py:send():234] send: files
2022-03-14 16:02:44,014 INFO    SenderThread:7212 [sender.py:_save_file():939] saving file wandb-metadata.json with policy now
2022-03-14 16:02:44,029 DEBUG   HandlerThread:7212 [handler.py:handle_request():130] handle_request: stop_status
2022-03-14 16:02:44,029 DEBUG   SenderThread:7212 [sender.py:send_request():248] send_request: stop_status
2022-03-14 16:02:44,035 INFO    MainThread:7212 [wandb_run.py:_console_start():1810] atexit reg
2022-03-14 16:02:44,038 INFO    MainThread:7212 [wandb_run.py:_redirect():1684] redirect: SettingsConsole.WRAP
2022-03-14 16:02:44,038 INFO    MainThread:7212 [wandb_run.py:_redirect():1721] Wrapping output streams.
2022-03-14 16:02:44,040 INFO    MainThread:7212 [wandb_run.py:_redirect():1745] Redirects installed.
2022-03-14 16:02:44,040 INFO    MainThread:7212 [wandb_init.py:init():633] run started, returning control to user process
2022-03-14 16:02:44,774 INFO    Thread-17 :7212 [dir_watcher.py:_on_file_modified():230] file/dir modified: D:\context_enhancement\context-main\combined\context_enhancement\wandb\run-20220314_160240-1u15e8i3\files\conda-environment.yaml
2022-03-14 16:02:44,774 INFO    Thread-17 :7212 [dir_watcher.py:_on_file_created():217] file/dir created: D:\context_enhancement\context-main\combined\context_enhancement\wandb\run-20220314_160240-1u15e8i3\files\wandb-metadata.json
2022-03-14 16:02:44,774 INFO    Thread-17 :7212 [dir_watcher.py:_on_file_created():217] file/dir created: D:\context_enhancement\context-main\combined\context_enhancement\wandb\run-20220314_160240-1u15e8i3\files\output.log
2022-03-14 16:02:44,972 INFO    Thread-23 :7212 [upload_job.py:push():137] Uploaded file C:\Users\Welcome\AppData\Local\Temp\tmpojbd0r0lwandb\1a3x5p24-wandb-metadata.json
2022-03-14 16:02:46,790 INFO    Thread-17 :7212 [dir_watcher.py:_on_file_modified():230] file/dir modified: D:\context_enhancement\context-main\combined\context_enhancement\wandb\run-20220314_160240-1u15e8i3\files\output.log
2022-03-14 16:02:48,801 INFO    Thread-17 :7212 [dir_watcher.py:_on_file_modified():230] file/dir modified: D:\context_enhancement\context-main\combined\context_enhancement\wandb\run-20220314_160240-1u15e8i3\files\output.log
2022-03-14 16:02:59,405 DEBUG   HandlerThread:7212 [handler.py:handle_request():130] handle_request: stop_status
2022-03-14 16:02:59,406 DEBUG   SenderThread:7212 [sender.py:send_request():248] send_request: stop_status
2022-03-14 16:03:13,089 DEBUG   SenderThread:7212 [sender.py:send():234] send: stats
2022-03-14 16:03:14,753 DEBUG   HandlerThread:7212 [handler.py:handle_request():130] handle_request: stop_status
2022-03-14 16:03:14,754 DEBUG   SenderThread:7212 [sender.py:send_request():248] send_request: stop_status
2022-03-14 16:03:30,131 DEBUG   HandlerThread:7212 [handler.py:handle_request():130] handle_request: stop_status
2022-03-14 16:03:30,132 DEBUG   SenderThread:7212 [sender.py:send_request():248] send_request: stop_status
2022-03-14 16:03:45,499 DEBUG   HandlerThread:7212 [handler.py:handle_request():130] handle_request: stop_status
2022-03-14 16:03:45,500 DEBUG   SenderThread:7212 [sender.py:send_request():248] send_request: stop_status
2022-03-14 16:03:46,357 DEBUG   SenderThread:7212 [sender.py:send():234] send: stats
2022-03-14 16:04:00,858 DEBUG   HandlerThread:7212 [handler.py:handle_request():130] handle_request: stop_status
2022-03-14 16:04:00,858 DEBUG   SenderThread:7212 [sender.py:send_request():248] send_request: stop_status
2022-03-14 16:04:16,207 DEBUG   HandlerThread:7212 [handler.py:handle_request():130] handle_request: stop_status
2022-03-14 16:04:16,208 DEBUG   SenderThread:7212 [sender.py:send_request():248] send_request: stop_status
2022-03-14 16:04:19,732 DEBUG   SenderThread:7212 [sender.py:send():234] send: stats
2022-03-14 16:04:31,563 DEBUG   HandlerThread:7212 [handler.py:handle_request():130] handle_request: stop_status
2022-03-14 16:04:31,563 DEBUG   SenderThread:7212 [sender.py:send_request():248] send_request: stop_status
2022-03-14 16:04:46,947 DEBUG   HandlerThread:7212 [handler.py:handle_request():130] handle_request: stop_status
2022-03-14 16:04:46,948 DEBUG   SenderThread:7212 [sender.py:send_request():248] send_request: stop_status
2022-03-14 16:04:52,970 DEBUG   SenderThread:7212 [sender.py:send():234] send: stats
2022-03-14 16:05:02,337 DEBUG   HandlerThread:7212 [handler.py:handle_request():130] handle_request: stop_status
2022-03-14 16:05:02,338 DEBUG   SenderThread:7212 [sender.py:send_request():248] send_request: stop_status
2022-03-14 16:05:17,704 DEBUG   HandlerThread:7212 [handler.py:handle_request():130] handle_request: stop_status
2022-03-14 16:05:17,705 DEBUG   SenderThread:7212 [sender.py:send_request():248] send_request: stop_status
2022-03-14 16:05:26,206 DEBUG   SenderThread:7212 [sender.py:send():234] send: stats
2022-03-14 16:05:33,069 DEBUG   HandlerThread:7212 [handler.py:handle_request():130] handle_request: stop_status
2022-03-14 16:05:33,070 DEBUG   SenderThread:7212 [sender.py:send_request():248] send_request: stop_status
2022-03-14 16:05:48,441 DEBUG   HandlerThread:7212 [handler.py:handle_request():130] handle_request: stop_status
2022-03-14 16:05:48,442 DEBUG   SenderThread:7212 [sender.py:send_request():248] send_request: stop_status
2022-03-14 16:05:59,462 DEBUG   SenderThread:7212 [sender.py:send():234] send: stats
2022-03-14 16:06:03,782 DEBUG   HandlerThread:7212 [handler.py:handle_request():130] handle_request: stop_status
2022-03-14 16:06:03,782 DEBUG   SenderThread:7212 [sender.py:send_request():248] send_request: stop_status
2022-03-14 16:06:19,130 DEBUG   HandlerThread:7212 [handler.py:handle_request():130] handle_request: stop_status
2022-03-14 16:06:19,130 DEBUG   SenderThread:7212 [sender.py:send_request():248] send_request: stop_status
2022-03-14 16:06:32,731 DEBUG   SenderThread:7212 [sender.py:send():234] send: stats
2022-03-14 16:06:34,473 DEBUG   HandlerThread:7212 [handler.py:handle_request():130] handle_request: stop_status
2022-03-14 16:06:34,474 DEBUG   SenderThread:7212 [sender.py:send_request():248] send_request: stop_status
2022-03-14 16:06:49,828 DEBUG   HandlerThread:7212 [handler.py:handle_request():130] handle_request: stop_status
2022-03-14 16:06:49,828 DEBUG   SenderThread:7212 [sender.py:send_request():248] send_request: stop_status
2022-03-14 16:07:05,172 DEBUG   HandlerThread:7212 [handler.py:handle_request():130] handle_request: stop_status
2022-03-14 16:07:05,172 DEBUG   SenderThread:7212 [sender.py:send_request():248] send_request: stop_status
2022-03-14 16:07:05,987 DEBUG   SenderThread:7212 [sender.py:send():234] send: stats
2022-03-14 16:07:20,651 DEBUG   HandlerThread:7212 [handler.py:handle_request():130] handle_request: stop_status
2022-03-14 16:07:20,652 DEBUG   SenderThread:7212 [sender.py:send_request():248] send_request: stop_status
2022-03-14 16:07:36,028 DEBUG   HandlerThread:7212 [handler.py:handle_request():130] handle_request: stop_status
2022-03-14 16:07:36,029 DEBUG   SenderThread:7212 [sender.py:send_request():248] send_request: stop_status
2022-03-14 16:07:39,277 DEBUG   SenderThread:7212 [sender.py:send():234] send: stats
2022-03-14 16:07:49,592 WARNING MsgRouterThr:7212 [router.py:message_loop():73] EOFError seen in message_loop
2022-03-14 16:07:49,748 INFO    MainThread:7212 [wandb_run.py:_atexit_cleanup():1780] got exitcode: 1
2022-03-14 16:07:49,749 INFO    MainThread:7212 [wandb_run.py:_restore():1752] restore
2022-03-14 16:07:49,891 INFO    SenderThread:7212 [sender.py:finish():1070] shutting down sender
2022-03-14 16:07:49,891 INFO    SenderThread:7212 [dir_watcher.py:finish():283] shutting down directory watcher
2022-03-14 16:07:50,124 INFO    SenderThread:7212 [dir_watcher.py:_on_file_modified():230] file/dir modified: D:\context_enhancement\context-main\combined\context_enhancement\wandb\run-20220314_160240-1u15e8i3\files\output.log
2022-03-14 16:07:50,125 INFO    SenderThread:7212 [dir_watcher.py:finish():313] scan: D:\context_enhancement\context-main\combined\context_enhancement\wandb\run-20220314_160240-1u15e8i3\files
2022-03-14 16:07:50,125 INFO    SenderThread:7212 [dir_watcher.py:finish():327] scan save: D:\context_enhancement\context-main\combined\context_enhancement\wandb\run-20220314_160240-1u15e8i3\files\conda-environment.yaml conda-environment.yaml
2022-03-14 16:07:50,125 INFO    SenderThread:7212 [dir_watcher.py:finish():327] scan save: D:\context_enhancement\context-main\combined\context_enhancement\wandb\run-20220314_160240-1u15e8i3\files\config.yaml config.yaml
2022-03-14 16:07:50,126 INFO    SenderThread:7212 [dir_watcher.py:finish():327] scan save: D:\context_enhancement\context-main\combined\context_enhancement\wandb\run-20220314_160240-1u15e8i3\files\output.log output.log
2022-03-14 16:07:50,131 INFO    SenderThread:7212 [dir_watcher.py:finish():327] scan save: D:\context_enhancement\context-main\combined\context_enhancement\wandb\run-20220314_160240-1u15e8i3\files\requirements.txt requirements.txt
2022-03-14 16:07:50,136 INFO    SenderThread:7212 [dir_watcher.py:finish():327] scan save: D:\context_enhancement\context-main\combined\context_enhancement\wandb\run-20220314_160240-1u15e8i3\files\wandb-metadata.json wandb-metadata.json
2022-03-14 16:07:50,136 INFO    SenderThread:7212 [dir_watcher.py:finish():327] scan save: D:\context_enhancement\context-main\combined\context_enhancement\wandb\run-20220314_160240-1u15e8i3\files\wandb-summary.json wandb-summary.json
2022-03-14 16:07:50,142 INFO    SenderThread:7212 [file_pusher.py:finish():177] shutting down file pusher
2022-03-14 16:07:50,143 INFO    SenderThread:7212 [file_pusher.py:join():182] waiting for file pusher
2022-03-14 16:07:50,359 INFO    WriterThread:7212 [datastore.py:close():281] close: D:\context_enhancement\context-main\combined\context_enhancement\wandb\run-20220314_160240-1u15e8i3\run-1u15e8i3.wandb
2022-03-14 16:07:51,034 INFO    Thread-34 :7212 [upload_job.py:push():137] Uploaded file D:\context_enhancement\context-main\combined\context_enhancement\wandb\run-20220314_160240-1u15e8i3\files\wandb-summary.json
2022-03-14 16:07:51,053 INFO    Thread-33 :7212 [upload_job.py:push():137] Uploaded file D:\context_enhancement\context-main\combined\context_enhancement\wandb\run-20220314_160240-1u15e8i3\files\requirements.txt
2022-03-14 16:07:51,056 INFO    Thread-31 :7212 [upload_job.py:push():137] Uploaded file D:\context_enhancement\context-main\combined\context_enhancement\wandb\run-20220314_160240-1u15e8i3\files\config.yaml
2022-03-14 16:07:51,071 INFO    Thread-32 :7212 [upload_job.py:push():137] Uploaded file D:\context_enhancement\context-main\combined\context_enhancement\wandb\run-20220314_160240-1u15e8i3\files\output.log
2022-03-14 16:07:51,080 INFO    Thread-30 :7212 [upload_job.py:push():137] Uploaded file D:\context_enhancement\context-main\combined\context_enhancement\wandb\run-20220314_160240-1u15e8i3\files\conda-environment.yaml
2022-03-14 16:07:51,975 ERROR   BackendThr:7212 [internal.py:wandb_internal():158] Thread HandlerThread:
Traceback (most recent call last):
  File "D:\Anaconda\envs\context_enhancement\lib\multiprocessing\connection.py", line 302, in _recv_bytes
    overlapped=True)
BrokenPipeError: [WinError 109] The pipe has been ended

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "D:\Anaconda\envs\context_enhancement\lib\site-packages\wandb\sdk\internal\internal_util.py", line 52, in run
    self._run()
  File "D:\Anaconda\envs\context_enhancement\lib\site-packages\wandb\sdk\internal\internal_util.py", line 99, in _run
    record = self._input_record_q.get(timeout=1)
  File "D:\Anaconda\envs\context_enhancement\lib\multiprocessing\queues.py", line 108, in get
    res = self._recv_bytes()
  File "D:\Anaconda\envs\context_enhancement\lib\multiprocessing\connection.py", line 216, in recv_bytes
    buf = self._recv_bytes(maxlength)
  File "D:\Anaconda\envs\context_enhancement\lib\multiprocessing\connection.py", line 321, in _recv_bytes
    raise EOFError
EOFError
2022-03-14 16:19:00,169 INFO    MainThread:7212 [wandb_run.py:_wait_for_finish():1912] got exit ret: None
2022-03-14 16:19:00,276 INFO    MainThread:7212 [wandb_run.py:_restore():1752] restore
2022-03-14 16:19:00,328 INFO    MainThread:7212 [internal.py:handle_exit():77] Internal process exited
